TestPlaybooks/playbook-Feed_Office365-Test.yml

```diff
@@ -86,7 +86,7 @@ tasks:
             value:
               complex:
                 root: Office365
-                accessor: Indicator.Value
+                accessor: Indicator.value
```

Comment:
You will need to update the test playbook to check against the raw_output instead of the entry context since we are getting rid of the entry context for the `get_indicators_command`.